Week2/homework/maartjes-work.js

```diff
@@ -42,24 +67,35 @@ const tuesday = [
   },
 ];
 
-const maartjesTasks = monday.concat(tuesday);
-const maartjesHourlyRate = 20;
+const durations = monday.duration.concat(tuesday.duration);
```

Comment:
This doesn't work. The `concat()` only works for arrays, but you're trying to apply it to an object. See [https://www.w3schools.com/jsref/jsref_concat_array.asp](url) for how to use it.